components/dashboard/src/projects/Prebuild.tsx

```diff
@@ -26,13 +26,20 @@ export default function () {
     const [prebuild, setPrebuild] = useState<PrebuildInfo | undefined>();
 
     useEffect(() => {
-        if (!team || !projectName || !prebuildId) {
+        if (!projectName || !prebuildId) {
             return;
         }
         (async () => {
+            const projects = (!!team
```

Comment:
this effect is bound to `[team]` and the early return is removed at L.29. 

I believe this might case a double call, if `setTeams` will return later due to slow network connection.

Reply:
Ah, could be. Good catch, thanks!

Reply:
Will add `!teams` to the early return `if`.